core-models/src/main/java/com/anytypeio/anytype/core_models/permissions/ObjectPermissions.kt

```diff
@@ -185,3 +218,28 @@ private val possibleToChangeLayoutLayouts = listOf(
     ObjectType.Layout.TODO,
     ObjectType.Layout.NOTE
 )
+
+private val fileLayouts = listOf(
+    ObjectType.Layout.FILE,
+    ObjectType.Layout.IMAGE,
+    ObjectType.Layout.VIDEO,
+    ObjectType.Layout.AUDIO,
+    ObjectType.Layout.PDF,
+)
+
+private val systemLayouts = listOf(
```

Comment:
let's remove this list, since it duplicates the existing list in SupportedLayout without any specific purpose